tests/pytests/test_aggregate_params.py

```diff
@@ -59,25 +59,26 @@ def test_group_by(self):
                                     ['brand', 'logitech', 'count', '35']], res)
 
 def test_apply(env):
-    env = Env(moduleArgs = 'DEFAULT_DIALECT 2')
-    conn = getConnectionByEnv(env)
-    env.flush()
-    env.expect('FT.CREATE', 'idx', 'PREFIX', 1, 'dkey', 'SCHEMA', 'name', 'TEXT', 'breed', 'TEXT', 'loc', 'GEO').ok()
-    waitForIndex(env, 'idx')
-    conn.execute_command('HSET', 'dkey:1', 'name', 'Lassie', 'breed', 'Rough Collie')
-    conn.execute_command('HSET', 'dkey:2', 'name', 'lessly', 'breed', 'Poodle')
-    conn.execute_command('HSET', 'dkey:3', 'name', 'Perrito', 'breed', 'poodle')
-    conn.execute_command('HSET', 'dkey:4', 'name', 'Lou Dog', 'breed', 'Dalmatian')
-    conn.execute_command('HSET', 'dkey:5', 'name', 'dipper', 'breed', 'dalmatian')
-    conn.execute_command('HSET', 'dkey:6', 'name', 'Duff', 'breed', 'Dalmatian')
-    conn.execute_command('HSET', 'dkey:7', 'name', 'Triumph', 'breed', 'Mountain Hound')
-    conn.execute_command('HSET', 'dkey:8', 'name', 'Chuck', 'breed', 'Saluki')
-    conn.execute_command('HSET', 'dkey:9', 'name', 'Tuk', 'breed', 'Husky')
-    conn.execute_command('HSET', 'dkey:10', 'name', 'Jul', 'breed', 'St. Bernard')
+    for dialect in [2, 5]:
+        env = Env(moduleArgs = 'DEFAULT_DIALECT {}'.format(dialect))
```

Comment:
Is there a test using `DEFAULT_DIALECT 5` to see automatic escaping is working (sanity)?

Reply:
Yes. `make pytest LOG=1 CLEAR_LOGS=0 SAN=addr QUICK=1` is running tests using `DEFAULT_DIALECT 5` (changed in [7fcc3f5](https://github.com/RediSearch/RediSearch/pull/4433/commits/7fcc3f5c8a2d020f74c706aa9b7386f9e2b59b93)):
- `test_tags:testTagAutoescaping`
- `test_json:testTagAutoescaping`

The following tests are using `DEFAULT_DIALECT 2` and `DEFAULT_DIALECT 5`
- `test_search_params:test_attr` 
- `test_search_params:test_binary_data`